dream_layer_backend/comfy_nodes/api_nodes/runway_text2img.py

```
@@ -0,0 +1,326 @@
+                else:
+                    # Unknown status
+                    print(f"[RunwayText2Img] Unknown status: {status}, continuing to poll...")
+                    time.sleep(poll_interval)
+                    
+            except httpx.HTTPStatusError as e:
+                if e.response.status_code == 404:
+                    raise RuntimeError(f"Task {task_id} not found")
+                else:
+                    raise RuntimeError(f"Error polling task status: {e.response.status_code}")
+            except httpx.TimeoutException:
+                print(f"[RunwayText2Img] Polling timeout, retrying...")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)